src/base64.js

```diff
@@ -32,11 +31,6 @@ module.exports = function base64 (alphabet) {
       return output
     },
     decode (input) {
-      if (url) {
-        input = input.replace('+', '-')
-        input = input.replace('/', '_')
-      }
-
```

Comment:
`Buffer.from('[base64url encoded str]', 'base64')` deals with `-` and `_` in a base64url encoded string so no need to do replacements prior to it.